src/task-status.js

```diff
@@ -0,0 +1,10 @@
+class Status {
```

Comment:
You don't need a function inside a class here, you can simply replace the function for something like this:
`const boxNotMarked = (box) => box.id === 'unmarked';

export default boxNotMarked;`